images/windows/scripts/build/Install-Git.ps1

```diff
@@ -4,18 +4,13 @@
 ##  Supply chain security: Git - checksum validation, Hub CLI - managed by package manager
 ################################################################################
 
-# Install the latest version of Git for Windows
-
-$downloadUrl = Resolve-GithubReleaseAssetUrl `
```

Comment:
By refusing to use a function, you lose some of the functionality, e.g. download retries. Instead of rewriting the script itself, it is better to fix the `Resolve-GithubReleaseAssetUrl` function